openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -41,13 +39,11 @@
 
 from common.djangoapps.entitlements.models import CourseEntitlement
 from common.djangoapps.student.models import (  # lint-amnesty, pylint: disable=unused-import
-    AccountRecovery,
     CourseEnrollmentAllowed,
     LoginFailures,
     ManualEnrollmentAudit,
     PendingEmailChange,
     PendingNameChange,
-    Registration,
```

Comment:
assuming both removed models are not being used?

Reply:
Yes. Optimized by Pycharm.